src/dispatch/incident/service.py

```diff
@@ -197,6 +197,14 @@ def create(*, db_session, incident_in: IncidentCreate) -> Incident:
         db_session=db_session,
         source="Dispatch Core App",
         description="Incident created",
+        details={
```

Comment:
Both incidents and cases can be created with specific statuses via the Web UI. I wonder if should also capture the original status when the incident or case was created with.